docs/guides/end-to-end-testing/testing-strategies/social-authentication.mdx

```diff
@@ -0,0 +1,423 @@
+    `Single-Page App` for you platform and`React` for the technology.
+  - Step 2: For `Social Connections`, please select Facebook, Microsoft, and
+    Google
+  - Step 3: Click try login and verify the connections are set up properly
+  - Step 4: You can either download the sample on the `Download Sample App`
+    button or clone it
+    [here](https://github.com/auth0-samples/auth0-react-samples/tree/d5848ed710db85d6ef060731b296603c0eca744b/Sample-01).
+    We will need this application in the next step. Select
+    `"I am Ready to Use Auth0"`. Your social connections on your Auth0 Tenant
+    should now be configured correctly
+
+### Setting up the Application
+
+On your Auth0 dashboard, visit the `Applications Dropdown` in the left menu.
+Select the dropdown and go to `Applications`
+
+- Select the `Default Application`
+- Visit the `Settings` tab
+  - Copy the `domain` and `client ID` fields inside `Basic Information`. These
+    will be needed shortly.
+  - The `Application Properties` should have `Single Page Application` selected
+    for `Application Type`,
+  - Inside `Application URIs`, please add `http://localhost:3000` to
+    `Allowed Callback URLs`, `Allowed Web Origins` and `Allowed Logout URLs`
+- Create the `src/auth_config.json` file from the
+  [`src/auth_config.json.example`](https://github.com/auth0-samples/auth0-react-samples/blob/d5848ed710db85d6ef060731b296603c0eca744b/Sample-01/src/auth_config.json.example)
+  file as described in the
+  [sample app](https://github.com/auth0-samples/auth0-react-samples/tree/d5848ed710db85d6ef060731b296603c0eca744b/Sample-01#create-an-api).
+- Once created, paste the `domain` and `client ID` into their respective fields
+  inside `src/auth_config.json`. The `audience` key can be removed. The
+  login/callback URIs will default to `http://localhost:3000` which is the url
+  of our test application.
+- [install](https://github.com/auth0-samples/auth0-react-samples/tree/d5848ed710db85d6ef060731b296603c0eca744b/Sample-01#project-setup)
+  the application dependencies and
+  [start the dev server](https://github.com/auth0-samples/auth0-react-samples/tree/d5848ed710db85d6ef060731b296603c0eca744b/Sample-01#compile-and-hot-reload-for-development)
+- Verify the application is stood up correctly by visiting
+  `http://localhost:3000` and clicking the blue `Log in` button in the top right
+  corner. Select one of our social connections and log in and make sure to
+  complete the `Authorize App` prompt. This only needs to be done once. This
+  should take you back to the sample app authenticated as the test user. When
+  finished, click the `Log Out` dropdown button. Repeat for Google, Microsoft,
+  and Facebook or the accounts you would like to test.
+
+Here is what this process should look like for Facebook setup:
+
+<DocsVideo src="https://vimeo.com/789093604"></DocsVideo>
+
+## Testing with Cypress
+
+Now that our application is set up and ready to go, we are ready to write our
```

Comment:
Does this section need more attention drawn to it?  Iconography or subheading perhaps?

Reply:
I am not sure. Usually with our other guides we tend to go with the H2 here, but I feel like in the others as well that the sections that matter somewhat get swallowed